frontend/src/pages/adminRoles.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { Button, Table } from 'reactstrap'
+import ChangeRole from '../components/changeRole'
+import ChangeYear from '../components/changeYear'
+import Head from '../components/head'
+
+import Nav from '../components/nav'
+
+class AdminRoles extends React.Component {
```

Comment:
What if I were to manually type adminRoles into the path? Doesn't seem like anything is stopping me from accessing the page as a member.